guides/v2.2/frontend-dev-guide/layouts/layout-overview.md

```diff
@@ -10,36 +10,34 @@ github_link: frontend-dev-guide/layouts/layout-overview.md
 functional_areas:
```

Comment:
remove the obsolete page parameters: `subgroup`, `menu_title`, `menu_order`, `menu_node`

Reply:
Fixed